pom.xml

```diff
@@ -93,6 +93,16 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
Do we know if this could break any other plugins due to conflicting dependencies?